File: PHP/Program/Mahasiswa.php
<!-- 
    Saya Mia Karisma Haq NIM [2102165] mengerjakan soal Latihan Praktikum-1 dalam mata kuliah DPBO
    untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin 
-->

<!-- Kelas Mahasiswa yang digunakan untuk merepresentasikan data Mahasiswa -->
<?php
    class Mahasiswa
    {
        // Atribut yang digunakan pada kelas Mahasiswa         
        private $nim = '';
        private $nama = '';
        private $prodi = '';
        private $fakultas = '';
        private $foto = '';
        
        // Constructor
        public function __construct($nim = '', $nama = '', $prodi = '', $fakultas = '', $foto = '')
        {
            $this->nim = $nim;
            $this->nama = $nama;
            $this->prodi = $prodi;
            $this->fakultas = $fakultas;
            $this->foto = $foto;
        }
        
        // Method set untuk set setiap atribut pada kelas mahasiswa
        public function setNim($nim)
        {
            $this->nim = $nim;
        }

        public function setNama($nama)
        {
            $this->nama = $nama;
        }

        public function setProdi($prodi)
        {
            $this->prodi = $prodi;
        }

        public function setFakultas($fakultas)
        {
            $this->fakultas = $fakultas;
        }

        public function setFoto($foto)
        {
            $this->foto = $foto;
        }
        
        // Method get untuk get setiap atribut pada kelas mahasiswa
        public function getNim()
        {
            return $this->nim;
        }

        public function getNama()
        {
            return $this->nama;
        }

        public function getProdi()
        {
            return $this->prodi;
        }

        public function getFakultas()
        {
            return $this->fakultas;
        }

        public function getFoto()
        {
            return $this->foto;
        }
        
        // Destructor
        function __destruct()
        {
            
        }
    }
?>

File: Java/Program/Mahasiswa.java
/* 
    Saya Mia Karisma Haq NIM [2102165] mengerjakan soal Latihan Praktikum-1 dalam mata kuliah DPBO
    untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin 
*/

// Kelas Mahasiswa yang digunakan untuk merepresentasikan data Mahasiswa
public class Mahasiswa {
    // Atribut yang digunakan pada kelas mahasiswa
    private String nim;
    private String nama;
    private String prodi;
    private String fakultas;
    
    // Constructor
    Mahasiswa()
    {

    }

    // Constructor : Overloading
    Mahasiswa(String nim, String nama, String prodi, String fakultas)
    {
        this.nim = nim;
        this.nama = nama;
        this.prodi = prodi;
        this.fakultas = fakultas;
    }

    // Method set untuk set setiap atribut pada kelas mahasiswa
    public void setNim(String nim) 
    {
        this.nim = nim;
    }

    public void setNama(String nama)
    {
        this.nama = nama;
    }
    
    public void setProdi(String prodi)
    {
        this.prodi = prodi;
    }

    public void setFakultas(String fakultas)
    {
        this.fakultas = fakultas;
    }

    // Method get untuk get setiap atribut pada kelas mahasiswa
    public String getNim()
    {
        return nim;
    }

    public String getNama()
    {
        return nama;
    }

    public String getProdi()
    {
        return prodi;
    }

    public String getFakultas()
    {
        return fakultas;
    }  
}
